pkg/services/ngalert/models/alertmanager.go

```diff
@@ -14,6 +14,12 @@ type AlertConfiguration struct {
 	OrgID                     int64 `xorm:"org_id"`
 }
 
+// HistoricAlertConfiguration represents a previously used alerting configuration.
+type HistoricAlertConfiguration struct {
+	AlertConfiguration `xorm:"extends"`
+	LastApplied        int64 `xorm:"last_applied"`
```

Comment:
```suggestion
    // LastApplied is the most recent time at which the configuration was applied to an Alertmanager, or 0 otherwise. Only set this field if the configuration has been applied by the caller.
	LastApplied        int64 `xorm:"last_applied"`
```

Reply:
...indentation may have gotten off in this suggestion. Might be easier to just apply in an editor rather than through github.